mmcv/runner/hooks/logger/dvclive.py

```diff
@@ -22,7 +22,10 @@ class DvcliveLoggerHook(LoggerHook):
         reset_flag (bool): Whether to clear the output buffer after logging.
             Default: False.
         by_epoch (bool): Whether EpochBasedRunner is used. Default: True.
-        kwargs: Arguments for instantiating `Live`_.
+        dvclive (Live, optional): An instance of the `Live`_ logger to use
+            instead of initializing a new one internally. Defaults to None. 
```

Comment:
```suggestion
        dvclive (Live, optional): An instance of the `Live`_ logger to use
            instead of initializing a new one internally. Defaults to None.
```